src/Mesh/babylon.abstractMesh.ts

```diff
@@ -1322,7 +1322,7 @@
             var index: number;
 
             // Action manager
-            if (this.actionManager) {
+            if (this.actionManager !== undefined && this.actionManager !== null) {

```

Comment:
Is it a proven optimization?

Reply:
this one seems fine:
https://jsperf.com/davca-vs-undefined
![image](https://user-images.githubusercontent.com/1306056/34463586-347873a2-ee15-11e7-99aa-2d4f8a6868bb.png)

Reply:
I see the same thing here. The results I see after a few test runs are consistent with what you see, and typically are even better for the undefined/null test.

While real code is much more complex than simple tests like on jsperf, it does appear to point in the direction that switching to strict equality did not have an impact, and it was the other if-statement changes I made. I didn't test each one individually but "optimized" them all at once then tested before and after.

Should I undo all if-statements which I just added strict equality too, and keep the changes where it is more undefined / null check?

Reply:
"Should I undo all if-statements which I just added strict equality too, and keep the changes where it is more undefined / null check?" ==> yes :)

Thank you for your help!